plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialCommandBlock.java

```diff
@@ -0,0 +1,83 @@
+package com.denizenscript.denizen.objects.properties.material;
+
+import com.denizenscript.denizen.objects.MaterialTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import org.bukkit.block.data.type.CommandBlock;
+
+public class MaterialCommandBlock implements Property {
+
+    public static boolean describes(ObjectTag material) {
+        return material instanceof MaterialTag
+                && ((MaterialTag) material).hasModernData()
+                && ((MaterialTag) material).getModernData().data instanceof CommandBlock;
+    }
+
+    public static MaterialCommandBlock getFrom(ObjectTag _material) {
+        if (!describes(_material)) {
+            return null;
+        }
+        else {
+            return new MaterialCommandBlock((MaterialTag) _material);
+        }
+    }
+
+    public static final String[] handledMechs = new String[] {
+            "conditional"
+    };
+
+    private MaterialCommandBlock(MaterialTag _material) {
+        material = _material;
+    }
+
+    MaterialTag material;
+
+    public static void registerTags() {
+
+        // <--[tag]
+        // @attribute <MaterialTag.conditional>
+        // @returns ElementTag(Boolean)
+        // @mechanism MaterialTag.conditional
+        // @group properties
+        // @description
+        // Returns whether this command block is conditional, or not.

```

Comment:
Many of these seem like they'd easily fit into `MaterialMode`.

Reply:
I'll merge this into `MaterialMode` in another set of changes I make later.